include/libc/assert.h

```diff
@@ -1,4 +1,6 @@
-#ifndef ASSERT_H
-#define ASSERT_H
+#ifndef LIBC_ASSERT_H
+#define LIBC_ASSERT_H
+
+void __assert(const char* file, int lineNum);
```

Comment:
I've actually looked at this, and what MM has named `__assert` doesn't look to be the real assert. In fact I don't think MM has the `__assert` function. It looks more like `LogUtils_HungupThread` from OoT.

Reply:
Yeah, I think you are right. At the same time this function looks like `_dbg_hungup` from DnM